master/cmd/determined-master/migrate.go

```diff
@@ -46,7 +46,7 @@ func runMigrate(cmd *cobra.Command, args []string) error {
 		}
 	}()
 
-	if _, err = database.Migrate(config.DB.Migrations, args); err != nil {
+	if _, err = database.Migrate(config.DB.Migrations, config.DB.DatabaseCode, args); err != nil {
```

Comment:
Don't mean to bikeshed, but I'd love a more specific term than "database code". I hear database code and I picture all the files containing queries, migrations, etc. Took me a minute to remember what this ticket referred to. Can't think of a great suggestion myself. In my head these are the "stateless" entities but that's probably not the right term.

Reply:
Maybe "ViewsAndTriggers", just to make the intended contents very clear. Calling it "stateless" or "transient" also suggests that you should be putting anything in there that can't get deleted and recreated routinely.

Reply:
renamed to views_and_triggers, its not 100% accurate but it probably is more clear the intention of it